x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/EnrichHelper.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.action;
+
+import org.elasticsearch.xpack.core.enrich.EnrichPolicy;
+import org.elasticsearch.xpack.esql.plan.logical.Enrich;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.elasticsearch.test.ESTestCase.randomBoolean;
+
+public class EnrichHelper {
+
+    static String randomEnrichCommand(EnrichPolicy policy, String name, Enrich.Mode mode) {
```

Comment:
This is better suited for `org.elasticsearch.xpack.esql.EsqlTestUtils`.

Reply:
++ I've moved it to https://github.com/elastic/elasticsearch/pull/104993/commits/6027453b5765b2bf0127613139dd3613e8f63da4